source/linux/Foundational_Components/Kernel/Kernel_Drivers/E5010_JPEG_Encoder.rst

```diff
@@ -389,7 +413,7 @@ with this clocking is as below :
 With these numbers theoretically E5010 core can handle 3840x2160@60fps equivalent load
 for 4:2:2 video formats and 3840x2160@75fps equivalent load for 4:2:0 video formats.
 
-This however requires the upstream element (for e.g. camera) to support above rates. On AM62A board fastest locally available upstream element source is `wave5 VPU decoder <../../../Foundational_Components_Multimedia_wave5.html>`__ which provides maximum performance of 3840x2160@59 fps with low bitrate files and we were able to achieve same performance after passing this decoded data to E5010 JPEG Encoder as shown in below example :
+This however requires the upstream element (for e.g. camera) to support above rates. On |__PART_FAMILY_DEVICE_NAMES__| board fastest locally available upstream element source is :doc:`wave5 VPU decoder <../../../Foundational_Components_Multimedia_wave5>` which provides maximum performance of 3840x2160@59 fps with low bitrate files and we were able to achieve same performance after passing this decoded data to E5010 JPEG Encoder as shown in below example :
```

Comment:
Please use `:ref:` instead of `:doc:`. It's less likely to break in the case of a structural change.

Reply:
Updated to use :ref: that was defined in Wave5 file